app/client/main.go

```diff
@@ -47,24 +48,32 @@ var consensusMod modules.ConsensusModule
 
 func main() {
 	var err error
-	consensusMod, err = consensus.Create(defaultConfigPath, defaultGenesisPath, true) // TECHDEBT: extra param required for injecting private key hack for debug client
+
+	runtime := runtime.New(defaultConfigPath, defaultGenesisPath, runtime.WithRandomPK())
+	cfg := runtime.GetConfig()
```

Comment:
I don't understand why all modules get passed `runtime` object (which contains config and genesis) and yet, the creation of the `Bus` takes the config and genesis objects separately. Please pass the runtime object to the `bus creation`

Reply:
Updated.

Since we removed `GetConfig()` and `GetGenesis()` from the `bus`, I am currently not passing in the `runtimeMgr` in 
https://github.com/pokt-network/pocket/pull/235/files#diff-91e861a0bd67a2298f7a42d2ad6f060e8b59153fab4519a9f6bfd4fa293919c4R75 
and also in the `CreateBus()`.

Please let me know if you think it could be beneficial have a reference to the runtime accessible via the bus. But anyway we can always consider adding it when/if needed